src/index.ts

```diff
@@ -205,7 +206,7 @@ export class CdktfProviderProject extends cdk.JsiiProject {
     });
     const upgradeScript = new CheckForUpgradesScriptFile(this, {
       providerVersion,
-      fqproviderName,
+      fqproviderName: fqproviderName,
```

Comment:
The prior syntax should've worked imo. What was the reason for this change?

Reply:
👍